jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -216,11 +216,23 @@
 		}
 		
 		Lookup(Class<?> lookupClass, Class<?> prevLookupClass, int lookupMode) {
-			this(lookupClass, prevLookupClass, lookupMode, true);
+			this(lookupClass, prevLookupClass, lookupMode,
+			/*[IF Java15]
+				false
+			/*[ELSE] Java15*/
+				true
+			/*[ENDIF] Java15*/
+			);
```

Comment:
Something like this may be clearer by putting the behaviour in a helper method like `lookupJLIPackageCheckDefault`
```suggestion
			this(lookupClass, prevLookupClass, lookupMode, lookupJLIPackageCheckDefault());
		}

/* For Java 15, the default is not to check for the "java.lang.invoke" package.
 * For earlier releases, these lookups are illegal
  */
static boolean lookupJLIPackageCheckDefault() {
	/*[IF Java15]
	return false;
	/*[ELSE] Java15*/
	return true;
	/*[ENDIF] Java15*/
}
```

Reply:
Agreed. Updated the PR.